Documentation/GeneralConventions/FileStructure.rst

```diff
@@ -191,61 +124,52 @@ overview. The TOC is built with the
         Usage/Index
         Contribution/Index
 
-The *maxdepth* property limits the depth of the page tree and *titlesonly*
-specifies that only the titles of the pages are displayed, no other headings.
-Both should emphasize the nature of an overview.
-
-The pages in the body of the directive are included in the TOC, and TOCs in
-those pages are resolved recursively. The documentation author must edit the
-list and provide a reST document at each of these file paths, interpreting the
-file paths as relative to the current file.
+For large pages it is advisable to limit the number of menu levels that will
+be displayed on the startpage by setting :rest:`:maxdepth:`. This will not
+limit the level of pages displayed in the main navigation.
 
-In general, the file hierarchy should match the menu hierarchy, and a page must
-be placed at either :file:`<page-name>.rst` or :file:`<page-name>/Index.rst`
-- the latter if subpages exist or are expected. For example, the page of the
-menu path "Usage > TYPO3 Backend > Shop Dashboard" should be stored at:
+For more information on possible options on the `toctree` directive see
+chapter :ref:`Toctrees in the reST reference <toctree>`.
 
-.. code-block:: none
-
-   .
-   └── Documentation
-       └── Usage
-           └── Typo3Backend
-               └── ShopDashboard.rst
+It is possible to use more then one `toctree` directive on the start page.
+Each `toctree` should have a :rst:`:caption:` in this case.
 
-or - if it has the subpage "Sell Widget" - at
+The main navigation will then be divided into sections labeled by those
+captions. The menu of the manual you are currently reading is an example for
+that:
 
-.. code-block:: none
+..  code-block:: rst
+    :caption: Documentation/Index.rst
 
-   .
-   └── Documentation
-       └── Usage
-           └── Typo3Backend
-               └── ShopDashboard
-                   ├── Index.rst
-                   └── SellWidget.rst
+    ..  toctree::
+        :hidden:
+        :caption: BASICS
```

Comment:
I would avoid uppercase words. If they should be uppercase by design, that should be done by CSS. If not, it is up to the user and we should not give that example.